ledger/src/blockstore.rs

```diff
@@ -518,6 +518,13 @@ impl BlockstoreRocksDbColumnFamilyMetrics {
     }
 }
 
+macro_rules! rocksdb_metric_header {
+    ($metric_name:literal, $cf_name:literal) => {
+        concat!($metric_name, ",cf_name=", $cf_name)
+    };
+}
```

Comment:
Out of curiosity, do you have any in mind right now? Looking ahead at the next change, I still see only "blockstore_rocksdb_cfs" as the metric name everywhere.

The macro is implicitly setup for metrics that are on a per-cf basis (with `cf_name" as a param), so wouldn't we want any additional metrics to fall under the same "blockstore_rocksdb_cfs" ?

Reply:
The performance metrics for read-query would be one example, which it will be done by using the [PerfContext](https://github.com/facebook/rocksdb/blob/main/include/rocksdb/perf_context.h) in RocksDB. (And it's also already available in rust-rocksdb as well [here](https://github.com/rust-rocksdb/rust-rocksdb/blob/master/src/perf.rs)!)

Since PerfContext itself already have many fields, it's probably cleaner that we use a different metric.